bluebottle/surveys/models.py

```diff
@@ -101,6 +224,7 @@ class Response(models.Model):
     submitted = models.DateTimeField(null=True, blank=True)
     project = models.ForeignKey('projects.Project', null=True, blank=True)
     task = models.ForeignKey('tasks.Task', null=True, blank=True)
+    user_type = models.CharField(max_length=200, blank=True, null=True)
```

Comment:
maybe we can be explicit about the possible values: (task_member, initiator, organization).